cartography/util.py

```diff
@@ -68,6 +68,19 @@ def timed(*args, **kwargs):
     return timed
 
 
+def increment(stat, count=1, rate=1):
```

Comment:
Can you add typehints here and maybe rename this to something clearer - what is actually being incremented here?

Reply:
Sure. Let me do that!

Reply:
Refactored